dc-build.sh

```diff
@@ -7,15 +7,15 @@ if [ $# -eq 0 ]
 then
     echo "Building docker compose"
     # Compose V2 integrates compose functions into the Docker platform,
-    # continuing to support most of the previous docker-compose features
+    # continuing to support most of the previous docker compose features
```

Comment:
The point of these comments is to point out that `docker compose` can be substituted for `docker-compose`. We could remove the comments entirely, but it doesn't make sense to remove the hyphen from them.

```suggestion
    # continuing to support most of the previous docker-compose features
```